types/theme.d.ts

```diff
@@ -175,6 +175,8 @@ export interface IThemeComponentStylesInput {
   ChatItem?: IComponentPartStylesInput
   ChatMessage?: IComponentPartStylesInput
   Divider?: IComponentPartStylesInput
+  Form?: IComponentPartStylesInput
```

Comment:
never have i ever update this file... I didn't even know about it; we should have a mechanism to do it for us automatically...

Reply:
Yep, this file was updated in some of the recent refactoring and adding typings for the styling. Agreed that there should be some mechanism for this.